core/src/test/java/org/apache/iceberg/rest/TestRESTCatalog.java

```diff
@@ -55,7 +61,7 @@ public void createCatalog() {
 
     this.restCatalog = new RESTCatalog((config) -> adaptor);
     restCatalog.setConf(conf);
-    restCatalog.initialize("prod", ImmutableMap.of());
+    restCatalog.initializeInternal("prod", ImmutableMap.of());
```

Comment:
Why does this call `initializeInternal` still? I think that normal initialization should be used for testing.

Reply:
No longer calls `initializeInternal`.